data-es2016plus.js

```diff
@@ -819,6 +819,7 @@ exports.tests = [
           firefox46: firefox.nightly,
           firefox51: firefox.developer,
           firefox53: firefox.sharedmem,
+          firefox55: true,
```

Comment:
Nope, this is for features that are present on nightly builds but the code is not on the released version (wrapped with #ifdef).  A flagged feature is present but disabled by default (which is the current situation for Firefox 53 and 54).  For Firefox 55, the feature will be enabled by default, which grants it a "true".